monitoring/microprofile-opentracing/src/test/java/io/quarkus/ts/microprofile/MicroProfileIT.java

```diff
@@ -37,7 +37,7 @@
     private static final int TIMEOUT_SEC = 59;
     private static final int POLL_DELAY_SEC = 10;
 
-    @JaegerContainer
+    @JaegerContainer(image = "jaegertracing/all-in-one:latest")
```

Comment:
1. Please, specify a version, so we will not be hit by sudden update. I assume, the latest is "jaegertracing/all-in-one:1.30"
2. Dockerhub has a download limit and breaking it regularly fails our pipelines. Could you create a copy of this image on quay.io?

Reply:
I believe quay.io already have the necessary image, `quay.io/jaegertracing/all-in-one:1.30`. Following the comment below, I will just adjust the version globally within https://github.com/quarkus-qe/quarkus-test-framework.